CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ### Fixed 
 - Fixed use of libYARP_robotinterface with YARP devices spawned by sensor plugins (https://github.com/robotology/gazebo-yarp-plugins/pull/544).
-
+- Fixed compilation against YARP 3.5 by removing spurious print in WorldInterface plugin (https://github.com/robotology/gazebo-yarp-plugins/pull/548).
```

Comment:
3.5? Did you mean 3.4?

Reply:
No, the problem of compilation is due to https://github.com/robotology/yarp/pull/2548 that will be released as part of YARP 3.5 .